README.md

```diff
@@ -29,6 +29,17 @@ A Clojure library for safely and reliably consuming SQS messages.
 (sqs/stop-consumer @consumer-id)
 ```
 
+### Region
```

Comment:
Maybe link to this section from the changelog?

Reply:
👍